great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -153,12 +153,17 @@ def __init__(
 
         self._usage_statistics_handler = usage_statistics_handler
 
-        # Necessary to annotate ExpectationSuite during `run()`
+        # Necessary to annotation ExpectationSuite during `run()`
```

Comment:
@Shinnnyshinshin Previous wording seemed better (unless you meant to "add annotation").  Thanks.

Reply:
This was just typo on my part :)